artemis/src/main/java/tech/pegasys/artemis/BeaconNode.java

```diff
@@ -40,6 +48,19 @@
 
 public class BeaconNode {
   private static final ALogger LOG = new ALogger(BeaconNode.class.getName());
+  private final Vertx vertx = Vertx.vertx();
+  private final ExecutorService threadPool =
```

Comment:
This might be a stupid question, but why is this not a static variable? I thought class level elements are static in Java

Reply:
static variables are not instance specific.  so if a change the value of a static variable it would be reflected in all cases where i do `new BeaconNode`